ci-operator/step-registry/fips-check/optional-operator-images-scan/fips-check-optional-operator-images-scan-commands.sh

```diff
@@ -130,17 +130,22 @@ for package in ${package_list}; do
         csv_test_result="${data_dir}/${package}/$csv"
         echo "##Starting scan for CSV '$csv'" >> $csv_test_result
         # retrieve related images from current CSV
-        image_list=$(cat /tmp/$catalog.json | jq -r '.items[].status.channels[]|select(.currentCSV=="'$csv'")|.currentCSVDesc.relatedImages[]' || true);
+        image_list_string=$(cat /tmp/$catalog.json | jq -r '.items[].status.channels[]|select(.currentCSV=="'$csv'")|.currentCSVDesc.relatedImages[]' || true);
```

Comment:
It should already been adressed in the csv part:
https://github.com/openshift/release/blob/e796e0731e9b50c098b50a7e4b187b1cbe9a9464/ci-operator/step-registry/fips-check/optional-operator-images-scan/fips-check-optional-operator-images-scan-commands.sh#L123

Reply:
Yes, but L123's `sort | uniq` is to filter out unique currentCSVs, we still need to use another `sort | uniq` in [L133](https://github.com/openshift/release/pull/56227/files#diff-0fb71f33d50ea5aaaf746676c4304b4b44a2f98658c402da08ad1eb812248f8dR133) to filter out unique related images.I didn't notice that before https://github.com/openshift/release/pull/56227#issuecomment-2331073946.

Current (each image appeared twice):

```
$ oc get packagemanifests.packages.operators.coreos.com -l catalog="qe-app-registry" -o json | jq -r '.items[].status.channels[]|select(.currentCSV=="'cert-manager-operator.v1.15.0'")|.currentCSVDesc.relatedImages[]'
brew.registry.redhat.io/rh-osbs/cert-manager-cert-manager-operator-bundle@sha256:b341...
registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:281b...
registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9@sha256:1d75...
registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:e351...
registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:3bdc...
brew.registry.redhat.io/rh-osbs/cert-manager-cert-manager-operator-bundle@sha256:b341...
registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:281b...
registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9@sha256:1d75...
registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:e351...
registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:3bdc...
```

If we add `sort | uniq`:
```
$ oc get packagemanifests.packages.operators.coreos.com -l catalog="qe-app-registry" -o json | jq -r '.items[].status.channels[]|select(.currentCSV=="'cert-manager-operator.v1.15.0'")|.currentCSVDesc.relatedImages[]' | sort | uniq
brew.registry.redhat.io/rh-osbs/cert-manager-cert-manager-operator-bundle@sha256:b341
registry.redhat.io/cert-manager/cert-manager-operator-rhel9@sha256:281b...
registry.redhat.io/cert-manager/jetstack-cert-manager-acmesolver-rhel9@sha256:1d75
registry.redhat.io/cert-manager/jetstack-cert-manager-rhel9@sha256:e351...
registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:3bdc...
```

That only happens if one CSV is the currentCSV for more than one channels. Like: 
- CSV `cert-manager-operator.v1.15.0` is the currentCSV for channel 'stable-v1' and 'stable-v1.15'.
- CSV `aws-load-balancer-operator.v1.1.1` is the currentCSV for channel 'stable-v1' and 'stable-v1.1'.

In these case it would cause duplicate image scanning.